src/components/Content/Content.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+function getRandomDigits() {
+  return Math.random().toString().slice(2);
+}
+
+type Props = {
+  items: string[];
+};
+export const Content: React.FC<Props> = ({ items }) => {
+  return (
+    <ul>
+      {items.map(item => (
+        <li
+          key={getRandomDigits()}
```

Comment:
these elements definitely won't be unique, in this case it is better to use `item` itself